data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py

```diff
@@ -26,12 +26,13 @@
 measurement
     - measurement_source_concept_id
     - measurement_concept_id
-death
-    - cause_source_concept_id
-    - cause_concept_id
 
 Remove those rows from the clean dataset
 Archive/sandbox those rows
+
+As of DC-1661, the death table has been removed.
+This allows the death table with suppressed cause_concept_id and
+cause_source_concept_id to persist without being deleted.9
```

Comment:
Small typo at end of sentence

```suggestion
cause_source_concept_id to persist without being deleted.
```

Reply:
Thanks for catching this, I updated it